backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -1422,7 +1431,8 @@ trait StandardAsyncExecutionActor
       } else {
         tryReturnCodeAsInt match {
           case Success(returnCodeAsInt)
-              if outOfMemoryDetected && memoryRetryRequested && !continueOnReturnCode.continueFor(returnCodeAsInt) =>
+              if outOfMemoryDetected && memoryRetryRequested && (!continueOnReturnCode.continueFor(returnCodeAsInt)
+                || !returnCode.continueFor(returnCodeAsInt)) =>
```

Comment:
It would be nice to encapsulate the interaction of `continueOnReturnCode` and the new `returnCodes` as much as possible. We can't get around supporting both, but every place we use them side-by-side will be a cause of confusion in the future. In this file, maybe we can add a new `continueForCode` method that checks both and returns the result.

Reply:
I believe this should be resolved by the below comment.